t/01-style.t

```diff
@@ -22,6 +22,7 @@ is qx{git grep -I -l '^use Test::\\(Exception\\|Fatal\\)' t/**.t}, '',
 is qx{git grep -I -l '^\\(throws\\|dies\\|lives\\)_ok.*\<sub\>' t/**.t}, '',
   'Only use simplified prototyped Test::Exception functions';
 is qx{git grep -I -l 'like.*\$\@' t/**.t}, '', 'Use throws_ok instead of manual checks of exceptions';
+is qx{git grep -I -l ' \\(if\\|unless\\) \$\@'}, '', 'Use try/catch instead of manual \$\@ checks';
```

Comment:
is there an empty space in the beginning of ' \\(if\\|unless\\) \$\@' intentionally or...?

Reply:
yes. Alternatively we could use `\<` to look for a word beginning but I wanted to look for the post-conditions only. I will consider if we generalize that to `\(if\|unless\).*$@`